atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/DbKeyValueServiceConfiguration.java

```diff
@@ -31,23 +31,28 @@
     public static final String TYPE = "relational";
 
     @Value.Default
-    public boolean oracleEnableEeFeatures() {
+    public boolean enableOracleEnterpriseFeatures() {
         return false;
     }
 
     @Value.Default
-    public int getRangeConcurrency() {
+    public int poolSize() {
```

Comment:
Not sure that poolSize is a good config value name, maybe something a little more explicit.  Maybe databaseConnectionPoolSize?